CONTRIBUTING.md

```diff
@@ -10,7 +10,21 @@ patches and features.
 
 ## Development environment
 
-Next steps should work on clear Ubuntu 18.04.
+## Mac OS 10.15
```

Comment:
Now instructions look slightly unstructured. I believe we need a general instruction with some cross links to specific notes. For example, `install necessary dependencies` and two notes for Ubuntu 18.04 and Mac 10.15 (in code blocks).

Reply:
Resonable